source/index.txt

```diff
@@ -95,11 +100,33 @@ What's New
 The :doc:`What's New </whats-new>` section lists new features and
 changes in each version.
 
-Additional Learning
--------------------
+Resources
+---------
+
+Visit the Developer Hub and MongoDB University for resources related to
+the MongoDB Java driver.
+
+Developer Hub
+~~~~~~~~~~~~~
+
+The Developer Hub contains content, resources, community, and social
+engagement for developers.
```

Comment:
1) I don't think the last two item of this list agrees with the verb "contains". 

2) It might be redundant to mention it contains content. Instead, why not mention what kind of content it contains (e.g. tutorials)?